feat: add zh_HK translations

18202781743 opened the pull request:
add zh_HK translations.

UTsweetyfish commented on debian/changelog:
```diff
@@ -1,3 +1,9 @@
+qt6-translations (6.8.0-1deepin) unstable; urgency=medium
```
```suggestion
qt6-translations (6.8.0-0deepin1) unstable; urgency=medium
```

UTsweetyfish commented on debian/changelog:
```diff
@@ -1,3 +1,9 @@
+qt6-translations (6.8.0-1deepin) unstable; urgency=medium
+
+  * * feat: add zh_HK translations
```
```suggestion
  * feat: add zh_HK translations
```

UTsweetyfish reviewed (changes requested):
请以 patch 的形式将文件放到 debian/patches

18202781743 committed:
debian: Add zh_HK translations and update zh_TW translations

- Add Hong Kong Traditional Chinese translation files as Debian patch
- Update Taiwan Traditional Chinese translation files
- Bump version to 6.8.0-0deepin1 in changelog

This follows Debian 3.0 (quilt) packaging format by managing
translation files as patches instead of direct modifications.

18202781743 commented:
https://pms.uniontech.com/bug-view-321739.html

UTsweetyfish commented:
lgtm, please integrate with `/integrate`.

18202781743 commented:
/integrate

18202781743 commented:
@UTsweetyfish 测试通过了，帮忙合入下，
